app/components/UserBets.tsx

```diff
@@ -202,16 +221,21 @@ export const UserBets = ({ fixedProductMarketMaker }: UserBets) => {
                         )}
                       >
                         <span>
-                          {tradedBalance} {collateralToken.symbol}
+                          {marketModel.isAnswerInvalid
+                            ? invalidMarketColleteralToRedeem
+                            : !canClaim && isResolved
+                              ? collateralSpent
+                              : tradedBalance}{' '}
+                          {collateralToken.symbol}
```

Comment:
[MINOR] [NON-BLOCKING]

I have the same opinionated and personal take here, feel free to use or dismiss the suggestion:

```tsx
function getResultAmount() {
  if (marketModel.isAnswerInvalid) return invalidMarketColleteralToRedeem;
  if (!canClaim && isResolved) return collateralSpent;

  return tradedBalance;
}

const resultAmount = getResultAmount();
```

And then inside the TSX side of the code we could do:
```tsx
<span>
  {resultAmount} {collateralToken.symbol}
</span>
```